.gitlab/container_build/docker_linux.yml

```diff
@@ -169,59 +169,62 @@ docker_build_fips_agent7_arm64_jmx:
     TAG_SUFFIX: -7-fips-jmx
     BUILD_ARG: --target test --build-arg FIPS_ENABLED=1 --build-arg WITH_JMX=true --build-arg WITH_JMX_FIPS=true --build-arg DD_AGENT_ARTIFACT=datadog-fips-agent-7*-arm64.tar.xz
 
-# build agent7 UA image
+# Build agent7 OTel image
 docker_build_ot_agent7:
   extends: [.docker_build_job_definition_amd64, .docker_build_artifact]
   rules:
     - !reference [.except_mergequeue]
     - when: on_success
   needs:
-    - job: datadog-ot-agent-7-x64
+    - job: datadog-agent-7-x64
+    - job: otel-agent-x64
   variables:
     IMAGE: registry.ddbuild.io/ci/datadog-agent/agent
     BUILD_CONTEXT: Dockerfiles/agent
     TAG_SUFFIX: -7-ot-beta
```

Comment:
Should we get rid of the `beta` suffix right now if it's supposed to be GA for 7.64?